src/Stratis.Bitcoin.Features.Miner.Tests/MinerSettingsTest.cs

```diff
@@ -1,99 +0,0 @@
-﻿using System;
```

Comment:
Can you keep these tests, but just remove the callbacks?

Reply:
Those are testing callback + inner logic (in case mining disabled- don't load other settings). 
Not sure if those are valid tests. But I don't mind keeping them if you'd like

Reply:
yeah please